tools/unit/src/main/kotlin/com/datadog/tools/unit/ObjectTest.kt

```diff
@@ -28,7 +28,7 @@ abstract class ObjectTest<T : Any> {
      * x.equals(x) is always true
      */
     @Test
-    fun `equals is reflexive`(forge: Forge) {
+    fun equalsIsreflexive(forge: Forge) {
```

Comment:
Why did you need to rename those methods ?

Reply:
We discussed on Slack about this. Is because this module is used by `integrationTests` module and the compiler doesn't understand this syntax.

Reply:
per this https://stackoverflow.com/a/62597742 backticked methods should work with settings we have, but people still report issues with 4.1.2, so it is better to use non-backticked names format imo. indeed it is (was) DEX limitation.